lib/core/src/display/camera/camera2d.rs

```diff
@@ -334,6 +349,11 @@ impl Camera2d {
     pub fn add_zoom_update_callback<F:ZoomUpdateFn>(&self, f:F) {
         self.rc.borrow_mut().add_zoom_update_callback(f);
     }
+
+    /// Adds a callback to notify when `screen` is updated.
+    pub fn add_screen_update_callback<F:ScreenUpdateFn>(&self, f:F) {
+        self.rc.borrow_mut().add_screen_update_callback(f);
+    }
```

Comment:
Here is one thing I really dislike about this solution and ideas how to fix it. `add_screen_update_callback` adds a callbkack and **removes existing callback if it was there**. It was acceptible for me in the case of `add_zoom_update_callback`, because the idea was that the zoom update callback is registered always only by the owner of the camera (the scene), however, this could be improved as well. The problem with `add_screen_update_callback` is bigger, because the callback is registered by some arbitrary code (navigator). Do we have any guarantees that we will not create something like minimap view which will register its own calllback and accidently unregister this one? Nope. So either we need to panic if the callback was already registered (ugly solution) or allow registering multiple callbacks (nice solution) - by using our callback manager here.